src/chainparams.cpp

```diff
@@ -365,7 +365,7 @@ class CMainParams : public CChainParams {
         fAllowMultiplePorts = false;
 
         nPoolMinParticipants = 3;
-        nPoolMaxParticipants = 5;
+        nPoolMaxParticipants = 20;
```

Comment:
You can't just bump it that easy - `CPrivateSendBroadcastTx::IsValidStructure()` is going to fail on old nodes https://github.com/dashpay/dash/blob/develop/src/privatesend/privatesend.cpp#L141 which in its turn will cause issues for dstx propagation https://github.com/dashpay/dash/blob/develop/src/net_processing.cpp#L2456 (miners running old version won't see/mine such dstx-es).